client/views/actions/topic_set_by.tpl

```diff
@@ -0,0 +1 @@
+Topic set by <a href="#" class="user" data-name="{{nick}}">{{mode}}{{nick}}</a> on {{localeDate when}}
```

Comment:
Both topic and time are sent as separate events from the server. Introducing "waiters" for this kind of stuff is a bit of an overkill.